src/components/author/author.twig

```diff
@@ -0,0 +1,23 @@
+<div class="c-author o-media{% if class %} {{class}}{% endif %}">
+  {% embed '@cloudfour/objects/bunch/bunch.twig' with { class: "o-media__object" } %}
+    {% block content %}
+      {% for author in authors %}
+        {% include '@cloudfour/components/avatar/avatar.twig' with { src: author.avatar() } only %}
+      {% endfor %}
+    {% endblock %}
+  {% endembed %}
+  <div class="o-media__content">
+    <div>
+      <span class="u-hidden-visually">Authored by</span>
+      {% for author in authors %}
+        {% if loop.last and loop.length > 1 %}and {% endif %}
+        <a href="{{ author.link() }}">{{ author.name() }}</a>
+        {%- if not loop.last and loop.length > 2 %},{% endif %}
+      {% endfor %}
+    </div>
+    <div>
+      <span class="u-hidden-visually">Published on</span>
+      <time datetime="{{ date | date("Y-m-d") }}">{{ date | date("F jS, Y") }}</time>
```

Comment:
The date gets read twice by VoiceOver so I got curious how others have handled this (both the `datetime` value and the string text is read). I found [this GH issue](https://github.com/bbc/simorgh/issues/1449) but it seems this is expected behavior. 

Leaving this note to document my findings but for now, it's not actionable as I was unable to find anything that helped confirm a different expected behavior.

Reply:
Good catch! I'm not sure the best way to handle that

Reply:
> I'm not sure the best way to handle that

No worries, I did a bit more research this AM but still wasn't able to find anything. It's okay to leave as-is. 👍